config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml

```diff
@@ -23,13 +23,15 @@ periodics:
     description: "Uses kubetest to run e2e tests (+Feature:GPUDevicePlugin) against a cluster created with cluster/kube-up.sh"
     testgrid-num-failures-to-alert: '6'
     testgrid-alert-stale-results-hours: '24'
+  decorate: true
```

Comment:
Can we delete the old test? the canary job below is working.

Reply:
@upodroid sure!